.github/workflows/build.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           - os: android
             runner: ubuntu-latest
           - os: ios
-            runner: macos-13
+            runner: macos-latest
```

Comment:
xcode version 15.0.1 not compatible macos-latest. Please check again when build iOS

Reply:
Relate: https://github.com/linagora/tmail-flutter/pull/2580

Reply:
Please also check release ios

Reply:
Need a tag build for that

Reply:
https://github.com/linagora/tmail-flutter/actions/runs/9298835002/job/25591485791 (Will cancel if build succeeds to prevent uploading new version)

Reply:
Looks like it's ok